app/helper_funcs.py

```diff
@@ -26,6 +26,7 @@ def get_rank_of_force(text):
     """
     model = FrankenBert('app\saved_model')
     text = clean_data(text)
+    print('cleaned text that is inputted to model: ', text)
```

Comment:
Don't forget to delete this print statement once we are done testing app deployment.

Reply:
good point. Will do!